src/Utility/Language.php

```diff
@@ -8,38 +8,74 @@
 class Language {
 
 	/**
-	 * @param string|null $languageList List of language codes or locales codes.
-	 * @param bool $forceLowerCase Flag to auto lower case country part of locale codes.
+	 * @param string|null $languageList List of locales/language codes.
+	 * @param array|bool|null $options Flags to forceLowercase or removeDuplicates locales/language codes
+	 *        @deprecated: Set to true/false to toggle lowercase
 	 *
 	 * @return array
 	 */
-	public static function parseLanguageList($languageList = null, $forceLowerCase = true) {
+	public static function parseLanguageList($languageList = null, array $options = []) {
```

Comment:
That's what I thought :P... 
I have already introduced bad typing by creating bad interfaces. But well, you merged it ;-).